### DISCUSSION TITLE
realtime subscription not working

### DISCUSSION THREAD

**Discussion by felibunnyy** [supabase/supabase#7820] [Questions] [answered]
Supabase realtime subscription not working as intended.
I have already done the necessary steps such as enabling replication for the table that requires listening to realtime changes and followed the documentation carefully but it still does not work.

Anyone has any idea on how to solve this?
![Screenshot 2022-07-21 at 1 50 43 AM](https://user-images.githubusercontent.com/105587981/180049565-0bf44191-0a95-4957-83d8-4fd4cdf417d8.png)

**Comment by Jannchie** [comment, marked as answer]
You may need to remove `React.StrictMode` from your web app. 

``` tsx
ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
  //<React.StrictMode>
    <App />
  //</React.StrictMode>
);
```

Currently, realtime does not seem to work properly in react strict mode. I have raised a related issue #7799 .

**Comment by w3b6x9** [comment]
@felibunnyy this is resolved in realtime-js/supabase-js v2 rc.

**Reply by felibunnyy** [reply]
thanks!
